static/app/components/replays/alerts/missingReplayAlert.stories.tsx

```diff
@@ -0,0 +1,6 @@
+import MissingReplayAlert from 'sentry/components/replays/alerts/missingReplayAlert';
+import storyBook from 'sentry/stories/storyBook';
+
+export default storyBook(MissingReplayAlert, story => {
+  story('All', () => <MissingReplayAlert orgSlug="MY-ORG" />);
```

Comment:
was this meant to be here?

Reply:
yup! simplest story ever, but i thought since we've gotta re visit this component, it could be helpful

Also, i'm finding the stories really helpful when i need to grab a screenshot for a PR